lib/HTTPLookupService.cc

```diff
@@ -191,98 +191,77 @@ Result HTTPLookupService::sendHTTPRequest(std::string completeUrl, std::string &
 
 Result HTTPLookupService::sendHTTPRequest(std::string completeUrl, std::string &responseData,
                                           long &responseCode) {
-    uint16_t reqCount = 0;
-    Result retResult = ResultOk;
-    while (++reqCount <= maxLookupRedirects_) {
```

Comment:
Here are more explanations for removing this loop.

This trivial loop is added just to print the redirect URL each time. For example, if `broker-1` redirects to `broker-2` and `broker-2` redirects to `broker-3`, there would be three logs before:

```
Curl [1] Lookup Request sent for http://broker-1...
Curl [2] Lookup Request sent for http://broker-2...
Curl [3] Lookup Request sent for http://broker-3...
```

After this patch, users won't know which brokers have been redirected to, the logs could be:

```
Curl Lookup Request sent for http://broker-1
```